src/sql/workbench/contrib/query/browser/gridPanel.ts

```diff
@@ -733,10 +734,22 @@ export abstract class GridTableBase<T> extends Disposable implements IView {
 		};
 	}
 
-	private rebuildActionBar() {
+	public rebuildActionBar() {
 		let actions = this.getCurrentActions();
 		this.actionBar.clear();
-		this.actionBar.push(actions, { icon: true, label: false });
+		if (this.options.displayActions ?? true) {
+			this.actionBar.push(actions, { icon: true, label: false });
+		}
+	}
+
+	public displayActionBar() {
```

Comment:
seems like you could add a property  showActionBar with boolean type instead of having 2 methods.

Reply:
Moved the action bar changes its own PR: https://github.com/microsoft/azuredatastudio/pull/16178